src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,33 @@
+package tests;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BaseTest {
+    protected static WebDriver driver;
+
+    JavascriptExecutor jse = (JavascriptExecutor) driver;{
+        jse.executeScript("scroll(0, 250);");
+    }
+    JavascriptExecutor jseHomePage = (JavascriptExecutor) driver;{
+        jseHomePage.executeScript("scroll(0, 250);");
+    }
+
+    public static WebDriver getDriver() {
```

Comment:
why you need this method? all tests have at as var available, e.g. HomePage homePage = new HomePage(driver);